pulp_container/app/tasks/sync_stages.py

```diff
@@ -602,7 +602,7 @@ async def create_signatures(self, man_dc, signature_source):
                 "signatures",
                 man_dc.content.digest,
             )
-            signatures_downloader = self.remote.get_downloader(url=signatures_url)
+            signatures_downloader = self.remote.get_noauth_downloader(url=signatures_url)
```

Comment:
You were supposed to switch from noauth_downloader to get_downloader here instead https://github.com/pulp/pulp_container/blob/main/pulp_container/app/tasks/sync_stages.py#L273

To explain: this is extensions api, meaning that these signature endpoints have same auth as the rest.
So with your changes line 605 will give you 401.